src/dpp/guild.cpp

```diff
@@ -122,6 +122,10 @@ guild_member& guild_member::set_communication_disabled_until(const time_t disabl
 	this->communication_disabled_until = disabled_timestamp;
 	return *this;
 }
+	
+bool guild_member::operator == (dpp::guild_member const& other_member) const {
```

Comment:
this is missing a comparison on guild id, so the same person in two guilds will be equal and shouldnt be.

Reply:
fair enough, I'll implement right away